Dockerfile

```diff
@@ -2,9 +2,11 @@
 # Unfortunately Dockerhub isn't as flexible as CircleCi or AWS Codebuild, so we have to hardcode this value here.
 # So if you in the future need to update this value, make sure you also edit the value in .ds_python_version.
 # These values should be kept in sync.
-ARG DS_PYTHON_IMG_VERSION=7.3.0
+ARG DS_PYTHON_IMG_VERSION=8.0.0
 
-FROM civisanalytics/datascience-python:${DS_PYTHON_IMG_VERSION}
+ARG PLATFORM=linux/x86_64
```

Comment:
I added this for consistency with datascience-python and civis-jupyter-r, to make it easier to test things locally on arm64 laptops